src/base/solvers/buildLPproblemFromModel.m

```diff
@@ -0,0 +1,58 @@
+function [ LPproblem ] = buildLPproblemFromModel( model)
+% Builds an COBRA Toolbox LP problem structure from a COBRA Toolbox model structure. 
+%
+% USAGE:
+%
+%    LPproblem = solveCobraLP(model)
+%
+% INPUT:
+%    model:     A COBRA model structure with at least the following fields
+%
+%                     * .S - The stoichiometric matrix
+%                     * .c - Objective coeff vector
+%                     * .lb - Lower bound vector
+%                     * .ub - Upper bound vector
+%
+% OPTIONAL INPUTS:
+%    model:     The model structure can also have these additional fields:
+%                     * .b the accumulation/depletion vector (default 0 for each metabolite).
+%                     * .osense - Objective sense (-1 means maximise (default), 1 means minimise)
+%                     * .csense - Constraint senses, a string containting the constraint sense for
+%                       each row in A ('E', equality(default), 'G' greater than, 'L' less than).
+%    
+% OUTPUT:
+%    LPproblem:       A COBRA LPproblem structure with the following
+%                     fields:
+%                     * .A - LHS matrix
+%                     * .b - RHS vector
+%                     * .c - Objective coeff vector
+%                     * .lb - Lower bound vector
+%                     * .ub - Upper bound vector
+%                     * .osense - Objective sense (-1 means maximise (default), 1 means minimise)
+%                     * .csense - Constraint senses, a string containting the constraint sense for
+%                       each row in A ('E', equality, 'G' greater than, 'L' less than).
+
+
+
+LPproblem.A = model.S;
```

Comment:
What if `model` has a field `A`?

Reply:
A (according to the definitions) should not be a model field. A is an LP specific field, and thus a field for an LPproblem. If some model contains A, it is already an LPproblem struct..